packages/grafana-prometheus/src/querybuilder/components/MetricCombobox.tsx

```diff
@@ -65,21 +71,59 @@ export function MetricCombobox({
     [getMetricLabels, onGetMetrics]
   );
 
+  const loadMetricsExplorerMetrics = useCallback(async () => {
+    const allMetrics = await onGetMetrics();
+    const metrics: string[] = [];
+    for (const metric of allMetrics) {
+      if (metric.value) {
+        metrics.push(metric.value);
+      }
+    }
+
+    return metrics;
+  }, [onGetMetrics]);
+
+  const metricsExplorerEnabled = config.featureToggles.prometheusMetricEncyclopedia;
+
   const asyncSelect = () => {
     return (
-      <Combobox
-        placeholder="Select metric"
-        width="auto"
-        minWidth={25}
-        options={loadOptions}
-        value={query.metric}
-        onChange={onComboboxChange}
-      />
+      <InputGroup>
+        <Combobox
+          placeholder="Select metric"
+          width="auto"
+          minWidth={25}
+          options={loadOptions}
+          value={query.metric}
+          onChange={onComboboxChange}
+        />
+
+        {metricsExplorerEnabled ? (
+          <Button
+            tooltip="Open metrics explorer"
+            aria-label="Open metrics explorer"
```

Comment:
do we need both here? 🤔 i think if you pass `tooltip` that supersedes `aria-label` anyway

Reply:
Yeah - IconButton does that, but not Button with an icon (which we want for the solid background)